app/javascript/controllers/index.js

```diff
@@ -30,3 +30,6 @@ application.register("articles-facet-paginate", ArticlesFacetPaginateController)
 
 import AccordionSectionController from "./accordion_section_controller"
 application.register("accordion-section", AccordionSectionController)
+
+import AutoCompleteHandlerController from "./auto_complete_handler_controller"
+application.register("auto-complete-handler", AutoCompleteHandlerController)
```

Comment:
Is there a reason you choose `auto-complete-handler` over something shorter like `auto-complete`?